charts/cryostat/README.md

```diff
@@ -1,5 +1,5 @@
 # Cryostat Helm Chart
-A Helm chart for deploying [Cryostat](https://cryostat.io/) on Kubernetes and OpenShift
+A Helm chart for deploying [Cryostat](https://github.com/cryostatio/cryostat) on Kubernetes and OpenShift
```

Comment:
I think the website is still probably the main landing page we want for users. This also deploys other components beyond the main Cryostat repo as well.